docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -231,6 +231,50 @@ docker run \
 
 The default Intel Quote Provider config is available in [Intel SGX Github repository](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/qcnl/linux/sgx_default_qcnl.conf).
 
+## Migrate from EPID Attestation to DCAP Attestation
+EPID attestation will be discontinued in 2025 and will no longer be available on
+any processors. All nodes using EPID attestation should migrate to DCAP
+Attestation.
+
+:::info
+
+Compute node operators, please hold for further instructions. We will coordinate
+the migration in phases to ensure that during the migration the network remains
+operational.
+
+:::
+
+:::info
+
+In case you are running AESM service on Docker please see [these instructions].
+
+:::
+
+To migrate to the DCAP Attestation run the following commands:
+1. See if your system [supports DCAP attestation].
+2. Stop your TEE nodes (Sapphire and Cipher).
+3. In case you are running AESM service on Ubuntu 22.04 run the following
+commands:
+  ```bash
+    sudo apt update
+    sudo apt remove libsgx-aesm-launch-plugin libsgx-aesm-epid-plugin
+    sudo apt install libsgx-aesm-ecdsa-plugin libsgx-aesm-quote-ex-plugin libsgx-dcap-default-qpl
+  ```
+4. Restart the aesmd.service:
+  ```bash
+  sudo systemctl restart aesmd.service
+  ```
+5. [Configure the Quote Provider].
+6. [Set up PCCS].
```

Comment:
Steps 5 and 6 are actually the same thing. "Set up PCCS" is one of the options of the "Configure the Quote Provider" step. Depending on the node operator's setup, they need to either: setup Intel PCS or setup PCCS of their cloud service provider, or setup their own PCCS. 

> The Intel Quote Provider (libsgx-dcap-default-qpl) needs to be configured in order to use either the Intel PCS, the PCCS of your cloud service provider, or your own PCCS. 

Please feel free to update the existing documentation there if this is not currently obvious.